convert sign in to mobile app guide to single page (#2654)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/guides/sign-into-mobile-app/main/index.md

### Configure the SDK

You need the following values from the Okta Application and the Admin Console that you worked with in the [Create an Okta application](#create-an-okta-application) section:

* **Client ID** &mdash; find it in the applications list or on the **General** tab of your app integration.
* **Okta Domain** &mdash; you can find the Okta Domain in the Admin Console's global header in the upper-right corner of the page. Click the section that displays your email and company name.  A drop-down box appears and displays general org information including the full Okta domain (for example, subdomain.okta.com).

> **Note:** Your Okta domain is different from your admin domain. Don't include `-admin` in your Okta domain.

You'll also need the full redirect URI that you defined in the [Define a callback route](#define-a-callback-route) section.

<StackSelector snippet="configuremid"/>

## Open the sign-in page

To sign in users, open a browser and display the Okta-hosted sign-in page.

To learn how to customize the sign-in page, see [Style the Widget](/docs/guides/style-the-widget/style-okta-hosted/).

<StackSelector snippet="signin"/>

## Get info about the user

> When a user signs in, their profile information (stored in Okta) is made available to your application. It's common to use this info to update your app's UI.

Reviewer: ```suggestion
When a user signs in, their profile information (stored in Okta) is made available to your application. You can use the user's profile information to personalize their app experience.
```

> By default, the profile items (called "claims") that Okta returns include the user's email address, first name, and last name. You can see the default items returned in the [/userinfo response example](/docs/reference/api/oidc/#response-example-success-5). The claims you see may differ depending on what scopes your app has requested (see <GuideLink link="../configure-packages/">Add and configure packages</GuideLink>).

Reviewer: ```suggestion
By default, the profile items (called "claims") that Okta returns include the user's email address, first name, and last name. You can see the default items returned in the [/userinfo response example](/docs/reference/api/oidc/#response-example-success-6). The claims you see may differ depending on what scopes your app has requested (see [Add and configure packages](#add-and-configure-packages).
```

<StackSelector snippet="getinfo"/>

<!-- You can also customize the items (called claims) that are returned from Okta. See [Token customization guide]. -->

## Check for a session at startup

When the app is closed and reopened, you need to check for the existence of an access token to see if the user has an existing session.

> <StackSelector snippet="checkfortoken"/>

Reviewer: Update the `<GuideLink>` reference in checkfortoken.md for ios.
